.github/workflows/build-test-deploy.yml

```diff
@@ -856,51 +856,49 @@ jobs:
-  #         installCommand: |
-  #           nix-channel --add ${{ env.NIX_NIXPKGS_CHANNEL }} nixpkgs
-  #           nix-channel --update
-  #           nix-env -iA nixpkgs.cachix nixpkgs.docker nixpkgs.docker-buildx
-  #         authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
-  #         name: composable-community
-  #     - uses: docker/login-action@v1
-  #       with:
-  #         username: ${{ secrets.DOCKER_HUB_USERNAME }}
-  #         password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
-  #     - run: |
-  #         # enable required features (see https://github.com/cachix/install-nix-action/issues/19)
-  #         mkdir -p ~/.config/nix
-  #         echo "system-features = kvm" >> ~/.config/nix/nix.conf
-  #     - run: |
-  #         cachix watch-exec --jobs 16 --compression-level $CACHIX_COMPRESSION_LEVEL composable-community nix -- build .#bridge-devnet-dali-container --no-update-lock-file --show-trace -L
-  #         docker load --input ./result
-  #         docker tag composable-centauri-devnet-container:latest "${{ env.DOCKER_REGISTRY_NAME }}/composable-centauri:latest"
-  #         docker push "${{ env.DOCKER_REGISTRY_NAME }}/composable-centauri:latest"
+  composable-bridge-devnet-container-publish:
+    if: ${{ github.ref == 'refs/heads/main' }}
+    name: "Docker composable-bridge-devnet"
+    needs:
+      - effects-gate
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - sre
+    concurrency:
+      group: ${{ github.workflow }}-composable-bridge-devnet-container-${{ github.ref }}
+      cancel-in-progress: true
+    container:
+      image: niteo/nixpkgs-nixos-22.05:316b762afdb9e142a803f29c49a88b4a47db80ee
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          echo "experimental-features = nix-command flakes" > /etc/nix/nix.conf
+          echo "sandbox = relaxed" >> /etc/nix/nix.conf
+          echo "narinfo-cache-negative-ttl = 0" >> /etc/nix/nix.conf
+      - uses: cachix/cachix-action@f5f67badd061acb62b5c6e25e763572ca8317004
+        with:
+          skipPush: true
+          installCommand: |
+            nix-channel --add ${{ env.NIX_NIXPKGS_CHANNEL }} nixpkgs
+            nix-channel --update
+            nix-env -iA nixpkgs.cachix nixpkgs.docker nixpkgs.docker-buildx
+          authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
+          name: composable-community
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - run: |
+          # enable required features (see https://github.com/cachix/install-nix-action/issues/19)
+          mkdir -p ~/.config/nix
+          echo "system-features = kvm" >> ~/.config/nix/nix.conf
+      - run: |
+          cachix watch-exec --jobs 16 --compression-level $CACHIX_COMPRESSION_LEVEL composable-community nix -- build .#bridge-devnet-dali-container --no-update-lock-file --show-trace -L
+          docker load --input ./result
```

Comment:
i would prefer it builds commit hash too non on main. people may need branches to test. there is example on how to do it in this file (actaully hat 100% cp paste with just change of package and container name)

Reply:
done.